src/skmultiflow/meta/online_under_over_bagging.py

```diff
@@ -263,27 +264,37 @@ def predict_proba(self, X):
         """
         proba = []
         r, c = get_dimensions(X)
-        try:
-            for i in range(self.actual_n_estimators):
-                partial_proba = self.ensemble[i].predict_proba(X)
-                if len(partial_proba[0]) > max(self.classes) + 1:
-                    raise ValueError("The number of classes in the base learner is larger than in the ensemble.")
 
-                if len(proba) < 1:
-                    for n in range(r):
-                        proba.append([0.0 for _ in partial_proba[n]])
-
-                for n in range(r):
-                    for l in range(len(partial_proba[n])):
-                        try:
-                            proba[n][l] += partial_proba[n][l]
-                        except IndexError:
-                            proba[n].append(partial_proba[n][l])
-        except ValueError:
-            return np.zeros((r, 1))
-        except TypeError:
+        if self.adwin_ensemble is None:
```

Comment:
Should not `self.ensemble` be used here, instead? (to cope with other ensemble models)

Reply:
Correct, fixed in 0d7aaeeb2841b9e5dbfe80fb821691ee2fbbd166